### PR TITLE
Allow attribute to be used, but not to show up in the schema

### PR DESCRIPTION
**Pull request by zeisler**
Docs for PR https://github.com/graphiti-api/graphiti/pull/157

### CODE REVIEW

File: guides/concepts/resources.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ attribute :first_name, :string
 {% include h.html tag="h4" text="2.1 Limiting Behavior" a="limiting-behavior" %}
 
 Each attribute consists of four flags: `readable`, `writable`,
```
Comment (richmolj) [inline guides/concepts/resources.md]: `s/four/five`

File: guides/concepts/resources.md [hunk 2]
```diff
@@ -96,6 +96,9 @@ attribute :name, :string, only: [:sortable]
 attribute :name, :string, except: [:writable]
 {% endhighlight %}
 
+The `schema` flag is not effected by `only/except` options.
```
Comment (timkrins) [inline guides/concepts/resources.md]: 'effected' here should be 'affected'

Comment (zeisler) [inline guides/concepts/resources.md]:
```suggestion
The `schema` flag is not affected by `only/except` options. 
```

**Comment by richmolj** [comment]
Thank you ❤️ !

**Pull request merged**